go.mod

```diff
@@ -3,18 +3,17 @@ module github.com/mendersoftware/deviceconnect
 go 1.14
 
 require (
-	github.com/gin-gonic/gin v1.7.4
+	github.com/gin-gonic/gin v1.7.7
```

Comment:
nit: This is not mentioned in the commit message

Reply:
Force-pushed to update the commit message, no other change.